content/webapp/components/ThemeWorks/index.tsx

```diff
@@ -50,6 +49,13 @@ const WorksCount = styled(Space).attrs({
   border-top: 1px solid ${props => props.theme.color('warmNeutral.300')};
 `;
 
+const WobblyEdgeWrapper = styled.div`
```

Comment:
mega np but all other similar things are called `WobblyEdgeContainer`. Should we decide to refactor `WobblyEdge` to have those containers in one place it'll be easier to find.

Very much np.